backend/src/main/resources/templates/Rhosak/actionRequiredBody.html

```diff
@@ -1,8 +1,9 @@
-{#include Rhosak/insightsEmailBody}
+{#include Rhosak/rhosakEmailBody}
 {#content-title}Action Required - {action.context.action_completion_time.toUtcFormat()}{/content-title}
 {#content-body}
 <tr>
     <td class="rh-content__block">
+        <div>Hello {user.firstName} {user.lastName},</div>
```

Comment:
Would love to confirm if this work, saw it as a possibility in the email template editor. It follows on my question on slack.
@gwenneg @josejulio